ocpp_codec/serializer.py

```diff
@@ -34,20 +34,25 @@ def _required_fields(dataclass_class) -> typing.List[dataclasses.Field]:
 
 
 def _run_validator(field: dataclasses.Field, data: typing.Any, *, parsing: bool) -> typing.Any:
-    # Fetch the validator to run
-    validator = field.metadata.get('validators', validators.noop)
-    if isinstance(validator, validators.BaseEncoder):
-        # Pick the right method based on whether we're parsing a message, or serializing a field
-        validator = validator.from_json if parsing else validator.to_json
+    # Fetch the validators to run
+    validator_list = field.metadata.get('validators', [validators.noop])
```

Comment:
```suggestion
    validator_list = field.metadata.get('validators', [])
```
I think an empty list works fine because it does the same thing than the following `validated_data = data` statement.

https://github.com/Polyconseil/ocpp-codec/blob/c5408459b374f133d2623f68ada75613e79b98ac/ocpp_codec/validators.py#L61-L63